packages/desktop/electron/watchers/active-window.js

```diff
@@ -45,20 +45,22 @@ export class ActiveWindowWatcher {
 	tracker() {
 		setInterval(async () => {
 			const activeWindow = await activeWin();
-
+			
 			if (activeWindow === undefined) return;
 
 			if (!this.app) {
 				this.startTime = Date.now();
 				this.app = activeWindow;
+				process.stdout.write(activeWindow.title);
 			}
 
 			//If the active window is changed store the used time data.
 			if (activeWindow.title !== this.app.title) {
 				this.storeTime();
 				this.app = null;
+				console.log("******WINDOW CHANGED*********");
 			}
-			console.log(activeWindow.title);
+			process.stdout.write(".");
```

Comment:
If we use `console.log` that runs exactly after this `.` is printed, that text won't be printed in a new line. Try to fix this. 
 
In case you haven't, set up the project, run the tracker, and let me know the bugs that you discover in this PR.

Reply:
Sir When the active window changes I am printing a blank console.log("") which give a "\n" character.

Reply:
> I am printing a blank console.log("") which will give a "\n" character

There's a cleaner solution.